mythical-creatures/test/fairy-test.js

```diff
@@ -0,0 +1,129 @@
+var assert = require('chai').assert;
```

Comment:
Have you driven out the tests in this file and `sphinx-test.js` to make sure that nothing is buggy?